azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeAudioSelectionChangedEvent.java

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.models;
+
+/**
+ * Event with audio selection changed.
+ */
+public final class CallCompositeAudioSelectionChangedEvent {
+    private final String selectionType;
```

Comment:
String/class naming is a bit vague, usage not clear? Can it be enum? can it be a class instance?

Audio is vague too, as it can be input/output, what is changed etc.